boards/cc2650stk/board.c

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 2016 Leon George
+ * (who won't go after you should you copy any part of this file ;-)
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+/**
+ * @ingroup     boards_cc2650stk
+ * @{
+ *
+ * @file
+ * @brief       Board specific implementations for the CC2650STK board
+ *
+ * @author      Leon M. George <leon@georgemail.eu>
+ */
+
+#include "board.h"
+#include "cpu.h"
+
+#include "periph/uart.h"
+
+#include "cc26x0-ioc.h"
+#include "cc26x0-gpio.h"
+#include "cc26x0-prcm.h"
```

Comment:
remove these 3 lines, `cpu.h` -> `cpu_conf.h` should take care of including all the vendor headers...